src/mrpro/operators/_LinearOperator.py

```diff
@@ -82,8 +83,22 @@ def adjoint(self, x: torch.Tensor) -> tuple[torch.Tensor,]:
         return (self._operator1.adjoint(x)[0] + self._operator2.adjoint(x)[0],)
 
 
-class LinearOperatorElementwiseProduct(LinearOperator, OperatorElementwiseProduct[torch.Tensor, tuple[torch.Tensor,]]):
-    """Operator elementwise multiplication with scalar/tensor."""
+class LinearOperatorElementwiseProductRight(
+    LinearOperator, OperatorElementwiseProductRight[torch.Tensor, tuple[torch.Tensor,]]
+):
+    """Operator elementwise right multiplication with scalar/tensor."""
```

Comment:
In the LinearOperator is the only time where you mention tensor and scalar as input. Nevertheless the signature only allows for tensors so I would suggest to remove scalar from the doc string.